source/_integrations/immich.markdown

```diff
@@ -36,6 +37,10 @@ Verify SSL certificate:
 
 The integration polls data every 60 seconds.
 
+## Media source
+
+A [media source](integrations/media_source) is provided for your user accessible [Immich](https://immich.app/) albums.
```

Comment:
```suggestion
A [media source](integrations/media_source) is provided for your [Immich](https://immich.app/) albums. It shows only the albums you own or that are shared with you.
```

Reply:
```suggestion
A [media source](integrations/media_source) is provided for your user accessible [Immich](https://immich.app/) albums.  It shows only the albums the user owns or that are shared with them.
```

I would replace the "you" by "the user" as you could have multiple immich integrations in HA. what do you think?

Reply:
we usually avoid speaking of "the user" and choose a more personal address. I rephrased my suggestion a bit to include more information. What do you think?

Reply:
sounds better 👍 let's start with this and see if there are complains about in the future 😄